admin/cordova_sandbox.sb

```diff
@@ -0,0 +1,106 @@
+(version 1) ; Fri Jan  6 12:01:53 2017
+
+; I M P O R T A N T:
+; /private/tmp/foobar below needs to be changed with the folder that holds the source code for Cordova project
+
+; (debug all)
+; (trace "trace.sb")
+
+; deny everything by default
+(deny default)
+
+; allow reading everything
+(allow file-read*)
+
+(allow sysctl-read)
+(allow network-outbound)
+(allow process-fork)
+(allow signal)
+
+(allow mach-lookup
+  (global-name "com.apple.cfprefsd.daemon")
+  (global-name "com.apple.system.opendirectoryd.membership")
+  (global-name "com.apple.system.notification_center")
+  (global-name "com.apple.system.opendirectoryd.libinfo")
+  (global-name "com.apple.CoreServices.coreservicesd")
+  (global-name "com.apple.system.logger")
+  (global-name "com.apple.cfprefsd.agent")
+  (global-name "com.apple.lsd.mapdb")
+  (global-name "com.apple.FSEvents")
+  (global-name "com.apple.DiskArbitration.diskarbitrationd")
+  (global-name "com.apple.diskmanagementd")
+  (global-name "com.apple.distributed_notifications@Uv3")
+  (global-name "com.apple.coreservices.quarantine-resolver")
+  (global-name "com.apple.FileCoordination")
+  (global-name "com.apple.SecurityServer")
+  (global-name "com.apple.backupd.sandbox.xpc")
+)
+
+; (allow file-write* (path "/private/var/folders/mg/fxfympln25j3qj4ysg8th3200000gn/T/ibn1YDAQAAAACnhgEA9QEAAPUBAAA-IBTOOLD-IN.Lx9ZIV6cRu"))
+(allow file-write* (subpath "/private/var/folders/mg"))
+
+; this is to be modified
+; (allow file-write* (path "/private/tmp/foobar/platforms/ios/build/emulator/HelloCordova.app/LaunchImage-700-Landscape@2x~ipad.png"))
```

Comment:
you can use "subpath" instead of "path":

`subpath "/private/tmp/foobar/platforms/ios/build/emulator/HelloCordova.app`